pkgs/top-level/all-packages.nix

```diff
@@ -2184,7 +2184,11 @@ in
 
   gh-ost = callPackage ../tools/misc/gh-ost { };
 
-  ghidra-bin = callPackage ../tools/security/ghidra { };
+  ghidra = callPackage ../tools/security/ghidra {
+    jdk = jdk11;
```

Comment:
Does it make any difference when you overwrite this at runtime? If not this can be removed.